docs/testing.md

```diff
@@ -0,0 +1,28 @@
+# Testing Open Angel Arena
+This is the continuation of [Install guide](/docs/install.md)
+Here is a guide how you can modify and run the OAA
+
+## Preparation
+You first need to open dev tools and load OAA project
```

Comment:
Should read:
"You first need to open 'Dota 2 Tools' and load OAA project". There is no such program called "Dev Tools", right? It's Dota 2 Tools.